src/components/RegionItem/RegionItem.tsx

```diff
@@ -26,12 +27,20 @@ const RegionItem: React.FC<{ region: Region }> = ({ region }) => {
   return (
     <StyledLink to={region.relativeUrl}>
       <Wrapper>
-        <CircleIcon $iconColor={iconColor} />
+        <IconContainer>
```

Comment:
I forgot to add tracking for clicks on the home page items, can you add some tracking?